LayoutTests/imported/w3c/web-platform-tests/webcodecs/chunk-serialization.any.js

```diff
@@ -72,3 +72,9 @@ async_test(t => {
 async_test(t => {
   runTest(t, 'video');
 }, 'Verify EncodedVideoChunk is serializable.');
+
+test(() => {
+  const chunk = createDefaultChunk("video", defaultVideoInit);
+  if (window.history)
+      assert_throws_dom("DataCloneError", () => history.pushState({ chunk }, null));
```

Comment:
The spec says a TypeError.

Reply:
We are changing this, see https://github.com/w3c/webcodecs/issues/589